libs/standard-tests/langchain_standard_tests/unit_tests/chat_models.py

```diff
@@ -138,6 +138,10 @@ def supports_anthropic_inputs(self) -> bool:
     def supports_image_tool_message(self) -> bool:
         return False
 
+    @property
+    def supported_usage_metadata_details(self) -> List[str]:
```

Comment:
List[Listeral[...]] so folks don't have to guess the names?

Reply:
smort